tests/integration/upstream_test.py

```diff
@@ -7,7 +7,7 @@
 
 class TestPolars:
     def test_dataframe(self) -> None:
-        import polars as pl
+        pl = pytest.importorskip("polars")
```

Comment:
To avoid using such constructs, may need to reconsider either the location of such tests (tests written for a specific implementation) or the procedure for running them.